scripts/scenes/card.gd

```diff
@@ -124,7 +132,8 @@ func show_decode_menu() -> void:
 	popup.popup()
 	
 func _on_decode_option_selected(id: int) -> void:
-	var decoder = get_node("/root/Main/Combat/DecoderManager")
+	var combat_scene = get_node("/root/Main/Combat")
```

Comment:
Gaat dit niet crashen in het geval dat de card in een andere scene dan de combat scene wordt geïnstantieerd?

Reply:
aka de deciphershop

Reply:
Ja, we gaan dit sowieso moeten herwerken denk ik, want momenteel staat de klik logica in de kaart, terwijl dit context afhankelijk is. In combat wil je de kaart spelen, in de shop wil je de kaart selecteren.

d.w.z. dat we de card scene een signal gaan moeten laten emitten, en dan in de omliggende context moeten we het connecten met het signal en de juiste gevolgen uitvoeren.